src/main/java/org/folio/service/pieces/flows/update/PieceUpdateFlowManager.java

```diff
@@ -116,7 +116,9 @@ protected Future<Void> updatePoLine(PieceUpdateHolder holder, RequestContext req
         }
         List<Piece> piecesToUpdate = List.of(holder.getPieceToUpdate());
         CompositePoLine poLineToSave = holder.getPoLineToSave();
-        poLineToSave.setReceiptStatus(calculatePoLineReceiptStatus(originPoLine, pieces, piecesToUpdate));
+        if (!poLineToSave.getReceiptStatus().equals(CompositePoLine.ReceiptStatus.CANCELLED)) {
```

Comment:
in other places we also include ongoing statuses, better to use the same utility method. Because in future it can be hard to understand if ongoing was omitted on purpose
For example
![image](https://github.com/user-attachments/assets/074e6d35-d06e-4273-80f1-1e2abd241bd9)
also log would be useful as well